walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,82 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    List<String> relevantAttendees = new ArrayList<>();
```

Comment:
Is this order of attendees important? ArrayList's `contains` method has complexity of O(n)

Reply:
Hmm... It is not important but I am not using `contains` on this array. Thus, I didn't think too much about it. 😅

Reply:
Yep because the focus of this PR is on algorithm, usage of data structure and time/space complexity is something that should be focused on. If the order of the attendees is not important, converting it into a list is just extra work that should be eliminated. `contains` is just one example of how improper choice of data structure might hurt you.

Reply:
I see. Thanks for sharing! Previously, I never thought that I should avoid using lists unnecessarily, as long as it doesn't worsen the performance (which in this case, it doesn't). Will take note of it in the future. (: